src/pages/study-room/components/header.tsx

```diff
@@ -0,0 +1,30 @@
+import { AppBar, Container, IconButton } from '@mui/material';
+import SettingsIcon from '@mui/icons-material/Settings';
+import ExitToAppIcon from '@mui/icons-material/ExitToApp';
+
+interface HeaderProps {
+  name: string;
```

Comment:
도메인 맥락이 조금 더 들어가주면 어떨까 싶기는 하네요!
name이라고만  표현되었을때 Header 컴포넌트가 어떤 이름을 받을지 헷갈리는 것 같아요.
```suggestion
interface HeaderProps {
  username: string;
```

Reply:
여기서 사용된 name은 방 제목을 의미하는데, 데이터를 받을 때부터 name으로 되어있어서 자연스럽게 해당 이름을 사용하게 됐네요. 이 부분은 title 등 다른 괜찮은 이름도 있을 것 같은데, 차후 의논해봐야 할 것 같습니다.